errors/errors.go

```diff
@@ -118,4 +118,18 @@ var (
 		Message:     "data encoding failure",
 		Description: "Failed to encode data. Please verify the encoding data.",
 	})
+
+	// ErrorCodeNotFound is returned when the requested resource is not found.
+	ErrorCodeNotFound = Register("errcode", ErrorDescriptor{
+		Value:       "NOT_FOUND",
```

Comment:
suggestion "RESOURCE_NOT_FOUND"  to be consistent with other errors

Reply:
Good call! Updated error codes.